fastify.js

```diff
@@ -26,6 +26,8 @@ const runHooks = require('./lib/hookRunner')
 
 const DEFAULT_JSON_BODY_LIMIT = 1024 * 1024 // 1 MiB
 
+function noop (anError, res) { }
```

Comment:
I don't think there's any reason for a no-op to have parameters.

Reply:
Agree.

Reply:
done